ProgressOnderwijsUtils.sln

```diff
@@ -13,6 +13,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.travis.yml = .travis.yml
 		appveyor.yml = appveyor.yml
 		How-to-push-to-nuget.txt = How-to-push-to-nuget.txt
+		install.ps1 = install.ps1

```

Comment:
is dit rare sln logica? zie dat het hierboven ook gebeurt, maar ziet er bijzonder uit

Reply:
Dit zijn de solution items. Die hebben niet echt functie, behalve dat je dan dingen makkelijker kunt openen vanuit Visual Studio.